scripts/restore-from-backup.sh

```diff
@@ -6,3 +6,12 @@ source .env
 MYSQL_CONTAINER=$(docker ps --format '{{ .Names }}' | grep mysql)
 docker cp $1 $MYSQL_CONTAINER:metakgp_wiki_db.sql
 docker exec $MYSQL_CONTAINER sh -c 'mysql -u metakgp_user -p'$MYSQL_PASSWORD' metakgp_wiki_db < metakgp_wiki_db.sql'
+
+result=$(find /backup -type f -name "images")
```

Comment:
Can you rename this to something more descriptive like images_exist? Also shouldn't the command be `find ./backup`?